delta/plugins/storage/src/test/resources/file/events/file-created.json

```diff
@@ -13,7 +13,7 @@
   "_rev" : 1,
   "_storage": {
     "@id": "https://bluebrain.github.io/nexus/vocabulary/disk-storage",
-    "@type": "DiskStorage",
+    "@type": "https://bluebrain.github.io/nexus/vocabulary/DiskStorage",
```

Comment:
any reason why this is no longer compacted?

Reply:
Yep. It is just like that.

Reply:
we're not constrained by compatibility with v1.4 because this field didn't exist... but yeah, we can do the api review and see.